packages/tooltip/stories/index.stories.mdx

```diff
@@ -72,25 +71,21 @@ You can easily change the placement of the content node relative to the invoker.
       ${tooltipDemoStyles}
     </style>
     <div class="demo-box-placements">
-      <lion-tooltip .config=${{ popperConfig: { placement: 'top' } }}>
+      <lion-tooltip .config=${{ popperConfig: { placement: 'top' } }} has-arrow>
         <button slot="invoker">Top</button>
         <div slot="content" class="demo-tooltip-content">Its top placement</div>
-        <lion-tooltip-arrow slot="arrow"></lion-tooltip-arrow>
       </lion-tooltip>
-      <lion-tooltip .config=${{ popperConfig: { placement: 'right' } }}>
+      <lion-tooltip .config=${{ popperConfig: { placement: 'right' } }} has-arrow>
         <button slot="invoker">Right</button>
         <div slot="content" class="demo-tooltip-content">Its right placement</div>
-        <lion-tooltip-arrow slot="arrow"></lion-tooltip-arrow>
       </lion-tooltip>
-      <lion-tooltip .config=${{ popperConfig: { placement: 'bottom' } }}>
+      <lion-tooltip .config=${{ popperConfig: { placement: 'bottom' } }} has-arrow>
         <button slot="invoker">Bottom</button>
         <div slot="content" class="demo-tooltip-content">Its bottom placement</div>
-        <lion-tooltip-arrow slot="arrow"></lion-tooltip-arrow>
       </lion-tooltip>
-      <lion-tooltip .config=${{ popperConfig: { placement: 'left' } }}>
+      <lion-tooltip .config=${{ popperConfig: { placement: 'left' } }} has-arrow>
         <button slot="invoker">Left</button>
         <div slot="content" class="demo-tooltip-content">Its left placement</div>
-        <lion-tooltip-arrow slot="arrow"></lion-tooltip-arrow>
       </lion-tooltip>
     </div>
   `}
```

Comment:
The code-block below doesn't have the `has-arrow` attribute.

Reply:
good catch